### ISSUE TITLE
Include gptfdisk in the Live CD

### ISSUE THREAD

**Issue by tom-rade** [void-linux/void-mklive#47]
Many people use gptfdisk when reparing or installing a Void Linux Installation. It is important that basic tools like these provided in gptfdisk, should be included in the base Live CD.

**Comment by chneukirchen** [comment]
Note that plain fdisk deals with GPT fine now (for years). What do you need gptfdisk for?

**Comment by tom-rade** [comment]
@leahneukirchen I personally use gdisk for wiping my GPT partition disk.

gdisk /dev/sda
x (expert mode)
z (zap)
Y (confirm)
Y (confirm)

**Comment by Duncaen** [comment]
there is also [wipefs(8)](https://man.voidlinux.org/wipefs.8) in `util-linux`, `cfdisk -z` does it too.

**Comment by atweiden** [comment]
After looking at this issue, I investigated replacing [voidvault](https://github.com/atweiden/voidvault)'s usage of sgdisk (gptfdisk) with sfdisk (fdisk).

Maybe someone can give me the `sfdisk` equivalent of this simple, readable `sgdisk` command:

```sh
sgdisk \
  --zap-all \
  --clear \
  --mbrtogpt \
  --new=1:0:+2M \
  --typecode=1:EF02 \
  --new=2:0:+550M \
  --typecode=2:EF00 \
  --new=3:0:0 \
  --typecode=3:8300
```

... because I can't find one. All the examples of `sfdisk` I've found are hideous by comparison.

**Comment by the-maldridge** [comment]
Sufficient tools are already on the iso to achieve this goal.  This is a wontfix for us.

**Comment by nathanielwheeler** [comment]
The "sufficient" tools provided are terrible for GPT partitioning.  You know what has been _specifically built_ for GPT partitioning?  `gptfdisk.`  

Not to mention that `gptfdisk` is no longer on the xbps repo list, so it has to be built with GCC.  I don't have space for this, as I'm trying to install Void off a dang CD.

**Comment by leahneukirchen** [comment]
You can install a few megabytes of packages fine using the live cd, did you try it?

It's also still in the repos, I don't see any issue here.

**Comment by nathanielwheeler** [comment]
I must have done something idiotic.  I've been trying to follow [this guide](https://wiki.voidlinux.org/Installation_on_UEFI,_via_chroot), since I need to be able to run more than 4 partitions on my desktop.  I used:
```
$ sudo su
# xbps-install gptfdisk
Unable to locate 'gptfdisk' in the repository pool.
```
I wish I had the log for the message, but I got an error message explaining that gptfdisk did not exist in the repository, or something to that effect.  After seeing your reply, I decided to give it another go:
```
$ sudo xbps-install gptfdisk
```
And it was found and installed.  I was flabbergasted, so I tried the previous method again, no repository pool.  So apparently su can't access the repository pool?  In that case, [this guide](https://wiki.voidlinux.org/Installation_on_UEFI,_via_chroot) probably needs to be updated to reflect that information.

**Comment by leahneukirchen** [comment]
You need to use `xbps-install -S gptfdisk`.

Anyway, you can create that partition table just as well using fdisk/cfdisk.

**Comment by nathanielwheeler** [comment]
I tried using `cfdisk` a couple of months back to create the partition tables, but Dell's boot would not pick them up no matter what I tried.  Some sort of CHS alignment issue I think.  

Since `gptfdisk` manipulates sectors instead of CHS values as `fdisk` does, I figured it would sidestep this issue.  I've already sunk too many hours in trying to get this distro working, and I'm not willing to risk a cylinder coordinate attribute misalignment making my Sunday a complete waste of time.
